cfpb/capital-framework-sandbox
# READY - Updated to show changes in cf-pagination

The atomic updates in cf-pagination (cfpb/capital-framework#326) will make breaking changes to existing markup examples. Updated the atomic documentation example but left the live cfgov-refresh example to show what needs to be changed in that project when it is updated to CFv4 in the fall.
## Changes
- Updated the markup example and rebuilt the styles using https://github.com/cfpb/capital-framework/pull/326
## Testing
- Checkout this branch locally
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- check http://localhost:3000/components/cf-pagination/
## Review
- @KimberlyMunoz 
- @cfarm 

[Preview this PR without the whitespace changes](?w=0)
## Screenshots

![screen shot 2016-07-13 at 2 56 05 pm](https://cloud.githubusercontent.com/assets/1280430/16817757/f06fb9f6-4909-11e6-9665-4293abd44912.png)
## Notes
- At the moment, this PR is for reviewing purposes only. **It should not be merged until the related Capital Framework PR is merged.** The styles that are currently built here will be overwritten when rebuilt without the locally linked package.
## Todos
- Migrate changes to cfgov-refresh
## Checklist
